errors/common.go

```diff
@@ -1,50 +1,15 @@
 package errors
 
 import (
-	stderrors "errors"
 	"fmt"
-
-	"github.com/pkg/errors"
-)
-
-// ABCI Response Codes
-// Base SDK reserves 0 ~ 99.
```

Comment:
Delete looks good.

I was wondering if we just remove the entire main.go and common.go files, leaving just errors.go with the new code. 

And move anything we need to errors.go to ensure all legacy code is gone

Reply:
@ethanfrey I'd like to do this in separate iteration so as not to disturb your stack trace PR.